docs/en/modules/ws2812.md

```diff
@@ -23,7 +23,15 @@ Initialize UART0 (TXD0) too if `ws2812.MODE_DUAL` is set.
 `ws2812.init([mode])`
 
 #### Parameters
-- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted) or `ws2812.MODE_DUAL`
+- `mode` (optional) either `ws2812.MODE_SINGLE` (default if omitted), `ws2812.MODE_SINGLE_OPEN`, or `ws2812.MODE_DUAL`
+
+WS2812-alikes generally expect 5V supply rails and when so powered may not
+consider the 3.3V logic high of the ESP8266 to be logic high. As such, one
+should consider using a level shifter.  However, the ESP8266 is 5V tolerant
+and supports open-drain GPIO, so a lower component-count option is to
+externally pull the WS2812 data line high with a resistor to 5V and have the
```

Comment:
Is that really necessary?  None of the other documentation, AFAIK, contains schematics, and there are modules that talk to things much more complicated than a WS2812.

The exact value of the resistor does not matter terribly much (though I think something ~10K is typical), but it does, in principle, vary with the design of the WS2812-alike's input circuitry.

In any case, would the following ASCII-art suffice?
```
5V >----[10K]---+
                |
GPIO4 >---------+------> WS2812 Data In
```